src/Microsoft.CodeAnalysis.BannedApiAnalyzers/Core/DocumentationCommentIdParser.cs

```diff
@@ -33,11 +33,10 @@ private static (string ParentName, string SymbolName)? ParseDeclaredId(string id
                 case 'M': // Methods
                 case 'P': // Properties
                 case 'T': // Types
-                    break;
                 case 'N': // Namespaces
+                    break;
```

Comment:
Please update the table at the end of https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.BannedApiAnalyzers/BannedApiAnalyzers.Help.md to add an example for banned namespace symbol and corresponding entry.

Reply:
Done.